src/test/java/org/tema6/CalculatorUnitTest.java

```diff
@@ -0,0 +1,63 @@
+package org.tema6;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import java.util.Scanner;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.tema6.Calculator.metricConvertor;
+
+class CalculatorUnitTest {
+
+    @ParameterizedTest
+    @CsvSource(value = {
+            "10km,10000000.0",
+            "5m,5000.0",
+            "2cm,20.0",
+            "7dm,700.0",
+            "15mm,15.0"
+    })
+    void testConvertMetric(String input, double expected) {
+        // given
+        final MetricConvertor metricConvertor = new MetricConvertor();
+
+        // when
+        final double result = metricConvertor.convertMetric(input);
+
+        // then
+        assertEquals(expected, result);
+    }
+
+    @Test
+    void simulateCalculatorTest() {
+        // given
+        Scanner scanner = new Scanner("10cm + 5mm");
```

Comment:
the behaviour of this scanner should be mocked
```suggestion
        Scanner scanner = mocks(Scanner.class);
        when(scanner.next()).thenReturn(...);
```